src/sharepoint/rest/tests/search.test.ts

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+import { expect } from "chai";
+import { Search } from "../search";
+
+describe("Search", () => {
```

Comment:
Yep, no worries on this - many of the classes are in a similar boat and we are all in need of ideas on getting stuff to work for in browser testing, search is even harder since we have to have things in the index...but will get it sorted at some point.